kikar_hamedina/facebook_feeds/fixtures/data_fixture_facebook_feeds.json

```diff
@@ -2870,7 +2870,7 @@
             "feed_type": "PP", 
             "persona": "30", 
             "is_current": true, 
-            "locally_updated": null
+            "locally_updated": "1970-01-01 00:00:00"
```

Comment:
A Facebook_Feed without locally_updated doesn't load properly on an MK page. I'm using this Facebook_Feed for the tests.

Best case would be if there was a test fixture, but that comes next.